### DISCUSSION TITLE
Translation issue

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [Stichoza/google-translate-php#185] [Q&A]
I am using ckeditor5 and when i copy and paste content in the textarea with large length it is responded by 500 error. I think there is a limit for text to be translated or i think wrong ? Please i need help also when i copy and paste formatted content i want the result to be transletd with same format. Pleas need help ASAP. Thanks in advance

**Comment by Stichoza** [comment]
Sorry for the super late answer, but in case anyone comes across this post: Yes, there is a limitation and it is described in [known limitations](https://github.com/Stichoza/google-translate-php#known-limitations) section in readme
